json-endpoints/jetpack/class.jetpack-json-api-plugins-endpoint.php

```diff
@@ -32,6 +32,26 @@
 		'action_links'    => '(array) An array of action links that the plugin uses.',
 	);
 
+	static $_response_format_v1_2 = array(
```

Comment:
why do we need to declare it here again?

Reply:
So that we have something to reference when we declare the endpoint

Reply:
could we remove the other and use this one?

Reply:
Don't we need both for backwards compatibility?